tests/test_dynamodb.py

```diff
@@ -59,6 +59,25 @@ class Config:
         }
 
 
+class Ticket(BaseModel):
```

Comment:
Nested models should probably not be `BaseModel` since they don't have their own tables. Use the  BaseModel from `pydantic` instead.

Reply:
done

Reply:
done